src/app/services/upload/upload-result.interface.ts

```diff
@@ -0,0 +1,9 @@
+interface UploadResult {
+  orginalName: string;
+  cleanName: string;
+  name: string;
+  extension: string;
+  size: number;
+  publicUrl: string;
+  filePath: string;
+}
```

Comment:
you could consider adding an id - this would make it easier to identify an upload